playbooks/server-install.yml

```diff
@@ -135,3 +136,64 @@
         key: "{{ keys }}"
         state: present
         exclusive: true
+
+    - name: uninstall docker if it exists
+      shell: |
+        for pkg in docker.io docker-doc docker-compose podman-docker containerd runc; do apt-get remove $pkg; done
+
+    - name: Create keyrings directory
+      file:
+        path: /etc/apt/keyrings
+        state: directory
+        mode: 0755
+
+    - name: Add Docker GPG apt Key
```

Comment:
There appears to be another way to do this more natively in ansible:

```
- name: Add an Apt signing key, uses whichever key is at the URL
  ansible.builtin.apt_key:
    url: https://ftp-master.debian.org/keys/archive-key-6.0.asc
    state: present
```

ref: https://docs.ansible.com/ansible/latest/collections/ansible/builtin/apt_key_module.html